integration/container/daemon_linux_test.go

```diff
@@ -0,0 +1,96 @@
+package container
```

Comment:
I'm not really sure where this test should go. It seems like it has more to do with daemon state than the container API/.

`s/daemon_linux_test/daemon_test.go/` and add a `skip.If()` because the relevant OS is where the daemon is running, not where the test client is running, so this would be skipped incorrectly if the client was windows, and the daemon was linux.

Reply:
This test will not compile on Windows and requires being on the same host since it's starts up a new daemon, kills processes, etc.

Reply:
> It seems like it has more to do with daemon state than the container API

It's kind of everything, sadly.